roles/kubernetes/node/templates/kubelet-config.v1beta1.yaml.j2

```diff
@@ -60,66 +60,23 @@ clusterDNS:
 - {{ dns_address }}
 {% endfor %}
 {# Node reserved CPU/memory #}
-{% if kube_reserved | bool %}
-kubeReservedCgroup: {{ kube_reserved_cgroups }}
+{% for scope in "kube", "system" %}
+{% if lookup('ansible.builtin.vars', scope + "_reserved") | bool %}
+{{ scope }}ReservedCgroup: {{ lookup('ansible.builtin.vars', scope + '_reserved_cgroup') }}
```

Comment:
Unless I am missing something I don't see these variables `{{scope}}_reserved_cgroup` defined anywhere.
Instead there is `kube_reserved_cgroups`, `system_reserved_cgroups`, with an s at the end.
This should have generated an error during testing?

Reply:
I agree, `lookup` is supposed to generate an error on non existing variable.
I think the most likely reason is that no testcases in the CI set `system|kube_reserved` to true.